ui/build/build.types.js

```diff
@@ -342,32 +358,38 @@ function writeIndexDTS (apis) {
     }
   })
 
+  const importName = []
   Object.keys(extraInterfaces).forEach(name => {
     if (extraInterfaces[name] === void 0) {
       // If we find the symbol as part of the generated Quasar API,
-      //  we don't need to import it from custom TS API patches
+      // we don't need to import it from custom TS API patches
       if (apis.some(definition => definition.name === name)) {
         return
       }
 
-      writeLine(contents, `import { ${name} } from './api'`)
+      importName.push(name)
+      // writeLine(contents, `import { ${name} } from './api'`)
```

Comment:
This can be removed, right?

Reply:
The comment, yes. I left it there and asked you for review because I wanted to make sure that was proper ts importing

Reply:
If I understand correctly, the new code keeps the references in an array and then import them all with a single statement, instead of adding one import per interface as I did before. So I think it's a good change